.github/workflows/release-post-merge.yml

```diff
@@ -225,3 +225,21 @@ jobs:
           asset_path: ./atlas-operator-all-in-one-${{ steps.tag.outputs.version }}.tar.gz
           asset_name: atlas-operator-all-in-one-${{ steps.tag.outputs.version }}.tar.gz
           asset_content_type: application/tgz
+
+      - name: Create SBOMs branch
+        env:
+          BRANCH: version-${{ steps.tag.outputs.version }}-sboms
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          make generate-sboms VERSION=${{ steps.tag.outputs.version }}
+          git checkout -b $BRANCH
+          git add .
+          git commit -m "Add SBOMs for version ${{ steps.tag.outputs.version }}"
+          git push --set-upstream origin $BRANCH
+
+      - name: Create SBOMs PR
+        uses: ./.github/actions/create-pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          REVIEWERS: igor-karpukhin,helderjs,josvazg,roothorp
```

Comment:
Improvement: move this to a GH env variable seeting to centralize when change is needed. There are 3 or 4 places with this already.

Reply:
that is a good idea, let me check the other usages

Reply:
done